docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -179,13 +180,15 @@ How the tile layer refines the visibility of tiles. One of the following:
 
 ##### `maxRequests` (Number, optional)
 
-The maximum number of concurrent `getTileData` calls. 
+The maximum number of concurrent `getTileData` calls.
 
-If `<= 0`, no throttling will occur, and `getTileData` may be called an unlimited number of times concurrently.
+If `<= 0`, no throttling will occur, and `getTileData` may be called an unlimited number of times concurrently regardless of how long that tile is or was visible.
 
-If `> 0`, a maximum of `maxRequests` instances of `getTileData` will be called concurrently. Additionally, for requests that are throttled, if the tile is no longer visible in the viewport when a request slot opens up, that request will be cancelled. When requests are not throttled, `getTileData` will be called for all visible tiles, regardless of how long that tile is or was visible.
+If `> 0`, a maximum of `maxRequests` instances of `getTileData` will be called concurrently. Requests may never be called if the tile wasn't visible long enough to be scheduled and started. Requests may also be aborted (through the `signal` passed to `getTileData`) if there are more than `maxRequests` ongoing requests and some of those are for tiles that are no longer visible.
 
-- Default: `8`
+If `getTileData` makes `fetch` requests, then `maxRequests` should correlate to the browser's maximum number of concurrent `fetch` requests. For Chrome, the max is 6.
```

Comment:
I'd specify "6 per domain", and "to increase this limit, utilize the `data` prop to support multiple domains".